test/with-packages/pkgs.nix/default.nix

```
@@ -137,7 +137,7 @@
           };
         };
       };
-  extras = hackage:
+  extras = _hackage:
```

```suggestion
  extras = hackage:
```
this file is automated too.